core/server/web/api/canary/admin/routes.js

```diff
@@ -167,14 +163,14 @@ module.exports = function apiRoutes() {
     router.post('/slack/test', mw.authAdminApi, http(apiCanary.slack.sendTest));
 
     // ## Sessions
-    router.get('/session', mw.authAdminApi, api.http(apiCanary.session.read));
+    router.get('/session', mw.authAdminApi, http(apiCanary.session.read));
     // We don't need auth when creating a new session (logging in)
     router.post('/session',
         shared.middlewares.brute.globalBlock,
         shared.middlewares.brute.userLogin,
-        api.http(apiCanary.session.add)
+        http(apiCanary.session.add)
     );
-    router.del('/session', mw.authAdminApi, api.http(apiCanary.session.delete));
```

Comment:
@rishabhgrg this change breaks the login flow :confused: needs a bit more digging I think

Reply:
@gargol Yeah noticed this change causing issues with tests breaking, looking into it. It seems `api.http` is doing something more which shared `http` is not covering, which is weird 😕

Reply:
the shared one is actually the one doing more :stuck_out_tongue:  It's wrapping stuff into a `Frame` instance